backend/middlewares/index.ts

```
@@ -1,25 +1,32 @@
+	return (req: Request, _res: Response, next: NextFunction): void => {
 		const { error } = schema.validate(req.body, { abortEarly: false });
 
 		if (error) {
-			res.status(400).json({
-				status: 'ERROR',
-				message: 'Validation Error',
-				details: error.details.map(detail => detail.message),
-			});
+			const validationError: ValidationError = createError(400, 'Validation Error');
+			validationError.details = error.details.map(detail => detail.message);
+			return next(validationError);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)